sdk/ml/azure-ai-ml/tests/component/e2etests/test_component.py

```diff
@@ -533,41 +533,6 @@ def test_command_component_create_autoincrement(self, client: MLClient, randstr:
         assert next_version_regex.match(next_component_asset.version)
         assert next_component_asset._auto_increment_version is False
 
-    @pytest.mark.disable_mock_code_hash
-    @pytest.mark.skipif(condition=not is_live(), reason="reuse test, target to verify service-side behavior")
-    def test_anonymous_component_reuse(self, client: MLClient, variable_recorder) -> None:
-        # component without code
-        component_name_1 = variable_recorder.get_or_record("component_name_1", str(uuid.uuid4()))
-        component_name_2 = variable_recorder.get_or_record("component_name_2", str(uuid.uuid4()))
-        component_resource1 = create_component(
-            client, _sanitize_python_variable_name(component_name_1), is_anonymous=True
-        )
-        component_resource2 = create_component(
-            client, _sanitize_python_variable_name(component_name_2), is_anonymous=True
-        )
-        assert component_resource1.id == component_resource2.id
-        assert component_resource1.environment == component_resource2.environment
```

Comment:
although the id should be not equal, while the environment and code compare should still be valid?